kong/plugins/apifortress/capturer.lua

```diff
@@ -0,0 +1,31 @@
+-- ©2016 API Fortress Inc.
+-- Captures the response payloads as they go through the wire
+local _M = {}
+
+local function read_response_body()
+	local chunk, eof = ngx.arg[1], ngx.arg[2]
+	local buffered = ngx.ctx.buffered
+	if not buffered then
+		buffered = {}
```

Comment:
this is very weird. Apparently the editor decided to screw everything up. I'm on it, sorry for the mess